server/src/logic.js

```diff
@@ -353,6 +361,9 @@ export const getHandlers = ({ models, creators: Creators }) => {
       user(response) {
         return response.getUser();
       },
+      eventLocations(response) {
```

Comment:
i think you should name this `destination` to match the schema. at the very least it should not be plural